### PR TITLE
Write lambdas using Scala tutorial

### PR DESCRIPTION
**Pull request by oilnam**

### CODE REVIEW

File: write-lambdas-in-scala.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+reating a Lambda with Scala <3
```
Comment (LATaylor-guardian) [inline write-lambdas-in-scala.md]: C*reating

File: write-lambdas-in-scala.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    - Add sbt assembly to your `plugins.sbt`
+    - Add the line `assemblySettings` to your `build.sbt`
+    - Run `sbt reload` to make the changes effective.
+    - You should now be able to run `sbt assembly` to produce your artifact.
+
+2. Upload your artifact to S3
+
+    - The first time you need to manually upload your artifact to S3. For CAPI, the path would be like
+    `content-api-dist/content-api/$STAGE/name-of-your-project/name-of-your-artifact.zip`. Adjust the path
```
Comment (LATaylor-guardian) [inline write-lambdas-in-scala.md]: I'm not sure much is gained from giving our path here?

**Comment by LATaylor-guardian** [comment]
Seems good to me 👍

File: write-lambdas-in-scala.md (hunk 2 above)
Comment (oilnam) [reply write-lambdas-in-scala.md]: I agree but a fake path is still better than no path 😅

**Review by piuccio — APPROVED** [review]

**Pull request merged**